questionGenerator.py

```diff
@@ -0,0 +1,272 @@
+import pandas as pd
+from utils import dataLoader, dataSaver, printAndLog
+from config import OpenAIBot
+import json
+
+
+class QuestionData:
+    """
+    Represents a class that handles question data generation.
+
+    Args:
+        config (Config): The configuration object.
+        videoData (VideoData): The video data object.
+        topicModeller (TopicModeller): The topic modeller object.
+        load (bool, optional): Whether to load existing question data. Defaults to True.
+    """
+
+    def __init__(self, config, videoData, topicModeller, load=True):
+        self.config = config
+
+        self.clusteredTopics = getClusteredTopics(videoData, topicModeller)
+        self.dominantTopics = getDominantTopic(self.clusteredTopics)
```

Comment:
Log top 10 or 5 topic returned in both cases. The object that is returned look complicated so logging relevant info is useful

Reply:
Thanks! Logging certain info while the process is running is useful as intermediate step